{"name": "test_right_way", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "c:\\Users\\alexi\\Downloads\\selenium.qa.studio-main\\tests\\test_shop.py:81: in test_right_way\n    ???\n        browser    = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0e302233cf4d65d89912368d8f0f53ce\")>\n        product    = <selenium.webdriver.remote.webelement.WebElement (session=\"0e302233cf4d65d89912368d8f0f53ce\", element=\"5f8cae43-6ffa-4b1e-881c-a8b1ea187c19\")>\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:95: in until\n    raise TimeoutException(message, screen, stacktrace)\nE   selenium.common.exceptions.TimeoutException: Message:\n        end_time   = 39119.25\n        message    = ''\n        method     = <function visibility_of_element_located.<locals>._predicate at 0x0000022FF6289120>\n        screen     = None\n        self       = <selenium.webdriver.support.wait.WebDriverWait (session=\"0e302233cf4d65d89912368d8f0f53ce\")>\n        stacktrace = None\n        value      = False"}, "description": "\n    Test case TC-4\n    ", "start": 1679680217604, "stop": 1679680230733, "uuid": "c8fe2a3e-aad1-4590-9a1e-337e6c65651a", "historyId": "832350b147fac251f172289a626c5cba", "testCaseId": "7b8323124234f69a8fde1e3de50f452e", "fullName": "tests.test_shop#test_right_way", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_shop"}, {"name": "host", "value": "Baribal"}, {"name": "thread", "value": "14340-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_shop"}]}